test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -889,6 +889,16 @@ class test_ucp_sockaddr : public ucp_test {
         return false;
     }
 
+    static ucs_status_t ep_flush(uct_ep_h ep, unsigned flags,
+                                 uct_completion_t *comp)
+    {
+        // UCT EP flush must be triggered only from discarding during the test
+        ucs_assert_always((comp != NULL) &&
+                          (comp->func ==
+                                   ucp_worker_discard_uct_ep_flush_comp));
```

Comment:
maybe just use ucs_empty_function_return_no_resource?
then no need to expose ucp_worker_discard_uct_ep_flush_comp